apps/trading/client-pages/referrals/tile.tsx

```diff
@@ -55,7 +55,7 @@ export const StatTile = ({
       >
         {title}
       </h3>
-      <div data-testid={`${testId}-value`} className="text-5xl text-left">
```

Comment:
Removing the value is what has broken the tests:
![image](https://github.com/vegaprotocol/frontend-monorepo/assets/133853298/58c56ab6-5da3-452b-a679-d098c3b81c3a)

`'[data-testid=combined-volume-value]': expected_volume,`
Are you happy for it to be added back in.